src/Forte.Optimizely.ContentUsage/Utils/TSGenerating/CamelcaseEnumStringGenerator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Reinforced.Typings;
+using Reinforced.Typings.Ast;
+using Reinforced.Typings.Generators;
+
+namespace Forte.EpiContentUsage.Utils.TSGenerating;
+
+public class CamelcaseEnumStringGenerator : EnumGenerator
+{
+    public override RtEnum GenerateNode(Type element, RtEnum result, TypeResolver resolver)
+    {
+        result = base.GenerateNode(element, result, resolver);
+
+        foreach (var enumValue in result.Values)
+        {
+            var value = enumValue.EnumValue;
+            enumValue.EnumValue = $"\"{char.ToLower(value[1])}{value.Substring(2)}";
```

Comment:
in other project we've been using `TypeBlueprint.ConvertToCamelCase` which comes with ReinforcedTypings, maybe it's worth to use it here as well?

Reply:
Yeah. But I decided to remove this generator completly. I cherry-picked some commits that I worked on a couple moths ago, but it seems to be completly useless for now.